### PR TITLE
chore: Remove print from code

### PR DESCRIPTION
**Pull request by philek**
Replaced with logger where useful output.
Removed everywhere else.

### CODE REVIEW

File: terrarun/server/__init__.py [hunk 1]
```diff
@@ -3809,7 +3797,7 @@ def _patch(self, current_user, current_job, tool_id):
             }.items()
             if request_attribute in attributes
         }
-        print(update_kwargs)
+        logger.debug("Patchinging terraform version: %s", update_kwargs)
```
Comment (MatthewJohn) [inline terrarun/server/__init__.py:3800]:
```suggestion
        logger.debug("Patching terraform version: %s", update_kwargs)
```

File: terrarun/server/__init__.py [hunk 2]
```diff
@@ -1977,11 +1969,10 @@ def check_permissions_get(self, current_user, current_job, run_id=None,):
     def _get(self, current_user, current_job, run_id=None):
         """Return run information"""
         if not run_id:
-            return {}, 404
+            return {"title": "Parameter run_id must be provided."}, 404
```
Comment (MatthewJohn) [inline terrarun/server/__init__.py:1972]: Did this format come from anywhere in particular?
If it's something you've decided, could you add a comment as such?
Every response format is currently based on API docs for Terraform Cloud

**Review by MatthewJohn — CHANGES_REQUESTED** [review]

File: terrarun/server/__init__.py (hunk 2 above)
Comment (philek) [reply terrarun/server/__init__.py:1972]: Haha, no. It's based on my mistaked interpretation of the docs. I should have nested it way more:
```{
  "errors": [
    {
      "status": "404",
      "title": "Parameter run_id must be provided."
    }
  ]
}```

Or used an ApiError like in other places. 
```{
  "errors": [ApiError(title = "Parameter run_id must be provided.", detail = "Parameter run_id must be provided.", status = 404)]
}```

But mostly it's not related to the print stuff so I shouldn't have committed it :)

File: terrarun/server/__init__.py [hunk 3]
```diff
@@ -816,7 +808,7 @@ def check_permissions_post(self, organisation_name, current_user, current_job, *
         """Check permissions"""
         organisation = Organisation.get_by_name_id(organisation_name)
         if not organisation:
-            print('NOT ORG FOND')
+            logger.error('Organisation "%s" not found.', organisation_name)
```
Comment (MatthewJohn) [inline terrarun/server/__init__.py:811]: Just because Yoda isn't fond of an organisation - doesn't mean it doesn't exist!

**Review by MatthewJohn — APPROVED** [review]

**Comment by MatthewJohn** [comment]
Merged in https://gitlab.dockstudios.co.uk/pub/terra/terrarun/-/merge_requests/76

**Pull request merged**